### PR TITLE
feat(agent): add sysctls option to builder

### PR DESCRIPTION
**Pull request by nandor-magyar**
I used the sysctls options to temporarily disable IPv6 for certain containers, they could also be useful in the future.

I was receiving a deprecation notice from docker-compose since a while now, I also removed the version addressing this warning.

### CODE REVIEW

File: web/crux-ui/docker-compose.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-version: '3.3'
```
Comment (polaroi8d) [inline web/crux-ui/docker-compose.yaml:1]: Why did you remove the version? There is no information in the PR about this.

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: web/crux-ui/docker-compose.yaml (hunk 1 above)
Comment (nandor-magyar) [reply web/crux-ui/docker-compose.yaml:1]: Because of deprecation, old versions of compose needed it, but recent version give a warning that it is a deprecated option.

Comment (polaroi8d) [reply web/crux-ui/docker-compose.yaml:1]: Okey, thanks for the clarification. (Missing from the PR description 🧒🏼)

**Review by polaroi8d — APPROVED** [review]
